server/README.md

```diff
@@ -1,6 +1,24 @@
 # hwapi server
 
 
+## Importing Data from C3
```

Comment:
I think this importing would be best done at image build time (a tool separate from the service would collate the data into a .sqlite, and that would then be copied into the image).